data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml

```diff
@@ -135,6 +135,17 @@
             <column name="update_actor" value="admin"/>
         </insert>
 
+        <insert tableName="config">
```

Comment:
We'd need a new liquibase file for this since this is coming after 5.0? Can someone else confirm this?

Reply:
Good point - if it goes into 5.1, it needs a new changeset, so ideally a new file so a liquibase upgrade would work.

Reply:
Good catch. @nkovacsx actually pointed out the need for this to be added in `ledger.utxo.1.1` instead, and I missed tha fact that it's coming in 5.1. I am not sure at all in terms of the versioning here. So it definitely needs to be in a new changeset.  But where does that changeset fall under in terms of versioning? (This change seems to be backwards compatible, so maybe different file under same version?)

Reply:
@kyriathar suggested to consult with @LWogan @JamesHR3, what do you guys think?

Reply:
What @JamesHR3 says below is right. 

Please disregards my above answer where I managed to blend the two together (i.e. the database update and the config schema update).